### PR TITLE
image-builder: log a warning msg when glitchtip not initialized

### PR DESCRIPTION
**Pull request by diaasami**
To help debug lack of issues showing up on glitchtip
https://issues.redhat.com/browse/HMS-1986

### CODE REVIEW

File: cmd/image-builder/main.go [hunk 1]
```diff
@@ -46,6 +46,9 @@ func main() {
 		if err != nil {
 			panic(err)
 		}
+
+	} else {
+		log.Warn("Sentry was not initialized")
```
Comment (ochosi) [inline cmd/image-builder/main.go:51]: How about adding a reference to the GlitchTip name as well?

**Review by ochosi — COMMENTED** [review]
The PR description is empty, but the change makes sense to me. Wondering if it would make sense to not only refer to Sentry but also GlitchTip, because someone less familiar with the subject may be looking for that in the logs.

**Review by croissanne — CHANGES_REQUESTED** [review]

File: cmd/image-builder/main.go [hunk 2]
```diff
@@ -46,13 +45,18 @@ func main() {
 		if err != nil {
 			panic(err)
 		}
+
```
Comment (croissanne) [inline cmd/image-builder/main.go:48]: Empty newline

File: cmd/image-builder/main.go [hunk 3]
```diff
@@ -17,7 +17,6 @@ import (
 	sentryecho "github.com/getsentry/sentry-go/echo"
 	"github.com/labstack/echo/v4"
 	"github.com/labstack/echo/v4/middleware"
-	"github.com/labstack/gommon/log"
```
Comment (croissanne) [inline cmd/image-builder/main.go:20]: We still need log to set the level.

**Review by croissanne — APPROVED** [review]

**Pull request merged**